src/hooks/useOAuthFlow.tsx

```diff
@@ -61,6 +61,14 @@ export const OAuthContextProvider = ({
     authConfig.usePKCE = true;
   }
 
+  // Ephemeral session is required
+  if (!authConfig.iosPrefersEphemeralSession) {
+    if (process.env.NODE_ENV !== 'test') {
+      console.warn('NOTE: LifeOmic requires PKCE. Overriding to usePKCE=true');
```

Comment:
Is PKCE the same as an ephemeral session or should this log statement be updated?

Reply:
🤦 thanks- Fix coming